commands/gitService.go

```diff
@@ -40,20 +40,33 @@ func (g *GS) UpdateGitViaGit(c *DeployCmd, response UploadResponse) error {
 	if err != nil {
 		return err
 	}
-	log.Println("[Debug] tempDir: ", tempDir)
+	log.Println("[DEBUG] tempDir: ", tempDir)
 	// Git objects storer based on memory
 	gitAuth := &gitHTTP.BasicAuth{
 		Username: "section-token", // yes, this can be anything except an empty string
 		Password: api.Token,
 	}
 	payload := PayloadValue{ID: response.PayloadID}
 	branchRef := fmt.Sprintf("refs/heads/%s",c.Environment)
-	r, err := git.PlainClone(tempDir, false, &git.CloneOptions{
-		URL:      fmt.Sprintf("https://aperture.section.io/account/%d/application/%d/%s.git", c.AccountID, c.AppID, appName),
-		Auth:     gitAuth,
-		Progress: os.Stdout,
-		ReferenceName: plumbing.ReferenceName(branchRef),
-	})
+	var r *git.Repository
+	if err != nil {
+		return err
+	}
+	if c.Quiet {
```

Comment:
The only thing we really need to change is what goes in the Progress field it looks like. Putting the entire thing in an if/else duplicates much more code than necessary. Something like:
```
p := os.Stderr
if c.Quiet { p = nil }
```
And then passing `p` to the git.PlainClone call would save many lines.